t/wit_status.t

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+. $(dirname $0)/regress_util.sh
+
+# Set up repo foo
+make_repo 'foo'
+foo_commit=$(git -C foo rev-parse HEAD)
+
+# Now set up repo bar to depend on foo
+mkdir bar
+git -C bar init
+echo "[{\"commit\":\"$foo_commit\",\"name\":\"foo\",\"source\":\"$PWD/foo\"}]" | jq '.' >> bar/wit-manifest.json
+git -C bar add -A
+git -C bar commit -m "commit1"
+bar_commit=$(git -C bar rev-parse HEAD)
+
```

Comment:
This looks good except that we generally guard the part before `set -x` with `prereq on` then `prereq off`, eg. https://github.com/sifive/wit/blob/5670099b45988e16c765ed696045231684de3b5d/t/wit_add_dep.t#L5

Reply:
Fixed by https://github.com/sifive/wit/pull/139/commits/bac3f1f533ffa7e20c3433efd82bb4ffb17f9213